matchmaking.py

```diff
@@ -199,10 +212,13 @@ async def check_for_match(bot: commands.Bot, user_id, min_rating, max_rating, mi
                 print(log_text)
                 with open("match_log.txt", "w") as file:
                     file.write(log_text)
-                await channel.send("We have a " + queue[user_id][
-                    "Game Type"] + " match! <@" + user_id + "> vs <@" + str(
-                    best_match) + ">. Find matches in <#" + str(
-                    BUTTON_CHANNEL_ID) + ">")
+                embed = discord.Embed()
+                embed.add_field(name=queue[user_id]["Game Type"] + " match found!",
+                                value=queue[user_id]["Name"] + " vs " + str(
+                                        queue[best_match]["Name"]) + "\n\n Find matches in <#" + str(
+                                        BUTTON_CHANNEL_ID) + ">")
+                await channel.send("<@" + user_id + "> <@" + str(
+                                        best_match) + ">", embed=embed)
```

Comment:
Not quite a review thing, but my suggestion for the match found message would be to stick a "vs." between the two users, make the "Superstars-On Match Found!" the embed title to increase the size & quicker to parse, and then drop the user vs user in the embed in favor of setting the "Find Matches in..." message as the description. Or heck, even make the message "**sleepy and poke found a match via # Matchmaking channel**" if you want to be cheeky.

Curious what others think but also acknowledge this is kind of splitting hairs.

Reply:
thanks for the feedback! I think I will role this out as is and if there is more community feedback on this we can update it to meet those needs